### ISSUE TITLE
False-positive: command-instead-of-module complains about `yum history`

### ISSUE THREAD

**Issue by mr1chter** [ansible/ansible-lint#4008]
##### Summary
I currently have a playbook that solely contains analysis/debugging tasks like 
```yaml
    - name: yum latest detail info
      ansible.builtin.shell:
        cmd: /usr/bin/yum history info
      register: yum_latest_detail_info

    - name: yum latest two transaction overview info
      ansible.builtin.shell:
        cmd: /usr/bin/yum history list | /usr/bin/head -n 5
      register: yum_latest_overview_info

```

ansible-lint complains that I should use the `yum` module here. But afaict this doesn't have support for `history` (or `info`, if someone uses that)

##### Issue Type

- Bug Report

##### OS / ENVIRONMENT

```console (paste below)
ansible-lint --version
ansible-lint 5.4.0 using ansible 2.10.8
```

- ansible installation method: OS package (ubuntu 22.04 WSL apt package `ansible`, version  `2.10.7+merged+base+2.10.8+dfsg-1`)
- ansible-lint installation method: OS package (same, `ansible-lint` version `5.4.0-2`

##### STEPS TO REPRODUCE

1. Create a playbook with above tasks

2. Run ansible-lint

(Not sure if this repro needs to be more detailed, I think the "bug" is pretty clear and is similar to https://github.com/ansible/ansible-lint/issues/369 . But I can elaborate if necessary)

##### Desired Behavior

Above tasks are not labeled as "command-instead-of-module" issues

##### Actual Behavior
It detects both tasks as "command-instead-of-module" issues.

**Comment by audgirka** [comment]
@mr1chter A PR to fix this would be welcome :)

**Comment by mr1chter** [comment]
Thank you very much!

I'll be sure to open a PR for the next issue :)
